plugin/src/main/java/com/denizenscript/denizen/utilities/Utilities.java

```diff
@@ -582,17 +582,33 @@ public static ElementTag enumLikeToLegacyElement(Object val) {
         throw new UnsupportedOperationException("Cannot get legacy name element, value isn't an enum: " + val);
     }
 
-    // TODO: need proper input backsupport, see https://discord.com/channels/315163488085475337/1011496047811506227/1301272242386370580
     public static <T> T elementToEnumlike(ElementTag element, Class<T> type) {
-        if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_21) && Keyed.class.isAssignableFrom(type)) {
-            return (T) Bukkit.getRegistry((Class<? extends Keyed>) type).get(parseNamespacedKey(element.asString()));
+        return elementToEnumlike(element, type, true);
+    }
+
+    @SuppressWarnings("unchecked")

```

Comment:
just disable unchecked warn in intellij, don't litter code with ide hints

Reply:
Removed